_layouts/nosection.html

```diff
@@ -15,8 +15,32 @@
   <link rel="icon" type="image/png" href="/images/favicon/16x16.png" sizes="16x16">
   <link rel="manifest" href="/manifest.json">
   <link rel="mask-icon" href="/images/favicon/safari-pinned-tab.svg" color="#dc0067">
+  <link rel="canonical" href="{{ site.url }}{{ page.url }}">
   <meta name="theme-color" content="#dc0067">
   <meta property="og:image" content="//bremen.freifunk.net/images/logo/logo_og_facebook.png" />
+  {% if page.layout == 'post' %}
+  <meta property="og:type" content="article" />
+  {% else %}
+  <meta property="og:type" content="website" />
+  {% endif %}
+  <meta property="og:url" content="{{ site.url }}{{ page.url }}" />
+  <meta property="og:description" content="{{ page.content | newline_to_br | strip_newlines | replace: '<br />', ' ' | strip_html | strip | xml_escape | truncate: 200 }}" />
```

Comment:
is ` | newline_to_br ` needed ?

Reply:
this construct is used to eliminate extra newlines which unfortunately can't be replaced via ```replace: '\n', ' '```
see https://stackoverflow.com/a/45147949/3484434

Reply:
okay, that is ugly - i also thought that `strip_newlines` makes dis job already ...